app/assets/stylesheets/jquery-ui.css

```diff
@@ -0,0 +1,1225 @@
+/*! jQuery UI - v1.11.4 - 2016-04-14
```

Comment:
I think this file is not required as you imported it above as `@import 'jquery-ui';`